chart/templates/_helpers.yaml

```diff
@@ -997,3 +997,27 @@ capabilities:
             app: keda-operator
   {{- end }}
 {{- end }}
+
+{{- define "kubernetesVersion" -}}
+{{ printf "%s" .Capabilities.KubeVersion.Version | trimPrefix "v" }}
+{{- end -}}
+
+{{- define "airflowKerberosContainerArgs" -}}
+{{- $kubectl_version := include "kubernetesVersion" . -}}
+{{- if and (semverCompare ">=2.8.0" .Values.airflowVersion) (semverCompare "<1.28" $kubectl_version) }}
```

Comment:
We need unit tests as well. But I wonder - why `Airflow >=2.8.0` ? We d o not seem to add any functionality for that?

Reply:
@potiuk Will work on adding the tests. Was down with a bad flu whole of last week :/

Hmm, I added 2.8.0 to avoid any regression i thought could occur due to airflow upgrades but now that I think of it, this is not a new feature, but an update to the existing behaviour. WDYT?

Reply:
Yes. No need to Add 2.8.0 in this case as we are not using any new feature added in `kerberos` command as I understand ?

Reply:
> Was down with a bad flu whole of last week :/

Keep healty . This is very important. :)

Reply:
Thank you. Taking care of myself :)